modules/firehose/main.tf

```diff
@@ -25,11 +25,11 @@ locals {
       url = "https://firehose-ingress.eu2.coralogix.com/firehose"
     }
   }
-  tags = {
+  tags = merge(var.user_supplied_tags, {
     terraform-module         = "kinesis-firehose-to-coralogix"
-    terraform-module-version = "v0.0.1"
+    terraform-module-version = "v0.0.8"
```

Comment:
It's not required, just a small suggestion.
IMHO would be better if this comes from a variable 😄